specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,228 @@
+MonochromaticOverlayPresenter
+===
+
+# Background
+
+The `MonochromaticOverlayPresenter` API in this spec is a Xaml Framework Element that lets you "draw" its background by
+pointing at another element; the rendering of that other element becomes the background of
+the Grid.
+
+`MonochromaticOverlayPresenter` element is similar to several precedents:
+* [SwapChainBackgroundPanel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SwapChainBackgroundPanel),
+which is a `Grid` that renders its background using a swap chain.
+* [CompositionMaskBrush](https://docs.microsoft.com/uwp/api/Windows.UI.Composition.CompositionMaskBrush),
+which  is a brush that gets its content from another (source) brush.
+* WPF's [VisualBrush](https://docs.microsoft.com/dotnet/api/System.Windows.Media.VisualBrush),
+which uses an element's rendering to draw a brush.
+
+`MonochromaticOverlayPresenter` is most similar to the WPF `VisualBrush`, the difference being that `MonochromaticOverlayPresenter`
+is an element rather than a Brush. So it can't for example be rendered as text foreground.
+
+This new `MonochromaticOverlayPresenter` will be used in the rendering of the 
+[DatePicker](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.DatePicker)
+control.
+
+# API Pages
+
+## MonochromaticOverlayPresenter class
+
+A [Framework Element](https://docs.microsoft.com/windows/winui/api/microsoft.ui.xaml.frameworkelement?view=winui-3.0-preview)
+whose background can be the rendering of another element, or using another element
+as a mask.
+
+```csharp
+public class MonochromaticOverlayPresenter : FrameworkElement
+{
+    public UIElement SourceElement { get; set; }
+    public Color ReplacementColor { get; set; }
+}
+```
+
+Set the `SourceElement` property to indicate which element's rendering should be drawn as the background of the `MonochromaticOverlayPresenter`. If the `Background` property is set, the SourceElement's rendering is on top of it. (The source is z-ordered on top of the background.
+
+Only the source element's rendering applies to the `MonochromaticOverlayPresenter`; the user
+cannot interact with it using the mouse, or move keyboard focus to it. However, `MonochromaticOverlayPresenter` will bring in system focus visuals, regardless of what the [UseSystemFocusVisuals](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Control.UseSystemFocusVisuals) property is set to. To avoid this, you should inset the margins to remove it.
```

Comment:
"bring in" is a big jargony. Perhaps

```suggestion
cannot interact with it using the mouse, or move keyboard focus to it. However, `MonochromaticOverlayPresenter` will render system focus visuals, regardless of what the [UseSystemFocusVisuals](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Control.UseSystemFocusVisuals) property is set to. To avoid this, you should inset the margins to remove it.
```

But even with that remark, I still am not sure what this is saying. Is it that it will render *its own* system focus visuals? Or that it will render system focus visuals of the source element? And whose margins should I inset? The MonochromaticOverlayPresenter or the SourceElement?

Reply:
My understanding is that it will render system focus visuals on itself, and you should inset the margins of the MonochromaticOverlayPresenter. @jevansaks is that correct?